src/Container.js

```diff
@@ -39,11 +39,12 @@ class Container {
    * @param {String} name
    * @param {Function} classname
    * @param {Array} dependencies
-   * @param {String} tag
+   * @param {String|Array} tags
    */
-  registerDefinition(name, classname, dependencies = [], tag = null) {
+  registerDefinition(name, classname, dependencies = [], tags = []) {
+    tags = typeof (tags) === 'string' ? [tags] : tags;
```

Comment:
agreed, but it's consistent with https://github.com/Elao/container.js/blob/master/src/Container.js#L139